### PR TITLE
Change styles set by codemirror default css to dark-mode color scheme

### PR DESCRIPTION
**Pull request by radfordwill**
… for wp code editors

### CODE REVIEW

File: codemirror.min.css [hunk 1]
```diff
@@ -0,0 +1 @@
...tant}.cm-s-default .cm-string{color:#ffb300!important}.cm-s-default .cm-string-2{color:#C39200!important}.cm-s-default .cm-qualifier{color:#9878b9!important}.cm-s-default .cm-meta{color:#ff5454!important}.cm-s-default .cm-builtin{color:#c5aefb!important}.cm-s-default .cm-bracket{color:#997}.cm-s-default .cm-tag{color:#170}.cm-s-default .cm-attribute{color:#00c}.cm-s-default .cm-hr{color:#999}.cm-s-default .cm-link{color:#7373d8!important}.cm-invalidchar,.cm-s-default .cm-error{color:#0025D4!important}.CodeMirror-matchingtag{background:rgba(255,150,0,.3)}.CodeMirror-activeline-background,.CodeMirror-activeline-gutter{background:rgba(109,189,255,.06)!important}.CodeMirror-gutter{background:#232323!important}.CodeMirror-wrap pre{background-color:#232323!important}.CodeMirror-selected{background:rgba(114,114,114,.46)}.CodeMirror-focused .CodeMirror-selected{background:rgba(114,114,114,.46)!important}.CodeMirror-line::selection,.CodeMirror-line>span::selection,.CodeMirror-line>span>span::selection{background:rgba(85,85,85,.4)!important}.CodeMirror-line::-moz-selection,.CodeMirror-line>span::-moz-selection,.CodeMirror-line>span>span::-moz-selection{background:rgba(85,85,85,.4)!important}
```
Comment (danieltj27) [inline codemirror.min.css]: Can you include a non-minified version of this please? If you could provide a Sass version that'd be great but a standard CSS version is fine as well.

Additionally, to make sure it's named in line with the main CSS, could you change the css file so it's called `dark_mode_editors` or something along those lines? Just so it's easy to tell what each CSS file does.

File: dark-mode.php [hunk 2]
```diff
@@ -126,7 +126,16 @@ public static function load_dark_mode_css() {
...
 			wp_enqueue_style('dark_mode');
-
+			
+			// Load style sheet only on pages affected by codemirror
+			global $pagenow;
+			
+			if ('plugin-editor.php' == $pagenow || 'widgets.php' == $pagenow || 'theme-editor.php' == $pagenow || 'customize.php' == $pagenow) {
+				
+			// change styles set by codemirror default css to dark-mode color scheme for wp code editors
+			wp_enqueue_style('dark_mode_codemirror', plugin_dir_url( __FILE__ ) . 'codemirror.min.css', array(), '1.2');
```
Comment (danieltj27) [inline dark-mode.php]: Can you update the version number here to `1.3` - also, it'd be worth making this stylesheet filterable like the main CSS file. I'm happy to merge this in though without a filter so we can get the editor CSS working first. Just need the version number sorted at this point.

**Comment by radfordwill** [comment]
[codemirror.zip](https://github.com/danieltj27/Dark-Mode/files/1529069/codemirror.zip)


Attaching both css files, de-minified and scss

github sees a conflict?? I will have to wait until I get a few moments to look at it closer. I updated the version and included both style sheet files. honestly don't see it at the moment, sorry for the mess.

**Comment by radfordwill** [comment]
Thanks for the offer for contributor list, 
.org username: willrad

**Comment by danieltj27** [comment]
Don't worry about it. We'll sort it. Worst case scenario is we close this PR and I can merge the code into the current code base and push it back into the repository. Might be easier that way as quite a lot of things have since changed.

Don't worry about it though. I'll add your name to the list in the next commit too. 👍

**Comment by danieltj27** [comment]
After some thought, I think the best course of action is to keep everything in one CSS file. After looking at the new changes, the colour variables can be taken from the main CSS file anyway as they're the set palette colours so it makes more sense I think.

What I'll do is close this PR but take the CSS from the change set and merge it in and tweak it so we can go from there. That should resolve some merge conflicts and give us a good base to go from.

**Comment by danieltj27** [comment]
I've added the Code Mirror styling in commit 03cf82d and also added you to the list of contributors. If you want to keep working on this then make a new PR with any new changes you have. Thanks for helping 👍

**Pull request closed without merging**